### PR TITLE
Migrated to traefik 2

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: infrastructure/docker/docker-compose.builder.yml [hunk 1]
```diff
@@ -3,8 +3,6 @@ version: '3'
 services:
     php-base:
         build: services/php-base
-        labels:
-            - "traefik.enable=false"
```
Comment (pyrech) [inline infrastructure/docker/docker-compose.builder.yml]: \o/

File: fabfile.py [hunk 2]
```diff
@@ -12,8 +12,8 @@
 env.project_name = 'app'
 # This is the host directory containing your PHP application
 env.project_directory = 'app'
-# This will be all your domain name, separated with comma
-env.project_hostnames = 'app.test'
+# This will be all your domain name, separated with comma. Don't forget backquote
```
Comment (pyrech) [inline fabfile.py]: Then, no longer separated by comma, are they?

File: infrastructure/docker/services/router/etc/traefik/traefik.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+global:
+  checkNewVersion: false
+  sendAnonymousUsage: false
+
+providers:
+  docker:
+    exposedByDefault: false
```
Comment (pyrech) [inline infrastructure/docker/services/router/etc/traefik/traefik.yaml]: :heart:

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.0.0
+
+* Switched to traefik 2
```
Comment (welcoMattic) [inline CHANGELOG.md]:
```suggestion
* Upgraded Traefik from 1.7 to 2.0
```

File: infrastructure/docker/services/router/etc/traefik/traefik.yaml (hunk 3 above)
Comment (welcoMattic) [reply infrastructure/docker/services/router/etc/traefik/traefik.yaml]: :heart_decoration:

File: fabfile.py (hunk 2 above)
Comment (lyrixx) [reply fabfile.py]: Indeed thanks

**Comment by lyrixx** [comment]
I fixed all your comments.Could someone test it locally to be really sure it works?

**Comment by lyrixx** [comment]
image

**Pull request merged**